### PR TITLE
Add typescript definitions for django-bananas.js

### PR DESCRIPTION
**Pull request by hbystrom91**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests)
- [x] The package does not already provide its own types, or cannot have its `.d.ts` files generated via `--declaration`
- [x] If this is for an npm package, match the name. If not, do not conflict with the name of an npm package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] Represents shape of module/library [correctly](https://www.typescriptlang.org/docs/handbook/declaration-files/library-structures.html)
- [x] `tslint.json` [should contain](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#linter-tslintjson) `{ "extends": "@definitelytyped/dtslint/dt.json" }`, and no additional rules.
- [x] `tsconfig.json` [should have](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#tsconfigjson) `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

File: types/django-bananas/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,192 @@
+// Project: https://github.com/5monkeys/django-bananas.js
```
Comment (martin-badin) [inline types/django-bananas/index.d.ts:1]:
```suggestion
// Type definitions for django-bananas 3.6
// Project: https://github.com/5monkeys/django-bananas.js
```

**Review by martin-badin — CHANGES_REQUESTED** [review]

**Comment by DangerBotOSS** [comment]
Inspecting the JavaScript source for this package found some properties that are not in the .d.ts files.
The check for missing properties isn't always right, so take this list as advice, not a requirement.
## django-bananas ([<kbd>unpkg</kbd>](https://unpkg.com/browse/django-bananas@latest/))
was missing the following properties:
1. useAdmin
1. Container
1. PageContext
1. usePage
1. LoginForm

<details>
<summary>as well as these 1 other properties...</summary>
<p>Translate</p>
</details>

<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against fc35065440d1904c9edf13b87518e10a7a15e6b1
</p>

File: types/django-bananas/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+    darth: any;
+    default: any;
+  }
+
+  const themes: themes;
+
+  export default themes;
+}
+// tslint:disable-next-line
+declare module 'django-bananas/colors' {
```
Comment (martin-badin) [inline types/django-bananas/index.d.ts:180]: please create separate files for modules

**Review by martin-badin — CHANGES_REQUESTED** [review]

File: types/django-bananas/tslint.json [hunk 3]
```diff
@@ -0,0 +1,3 @@
+{ 
+  "extends": "@definitelytyped/dtslint/dt.json"
+}
```
Comment (martin-badin) [inline types/django-bananas/tslint.json:3]:
```suggestion
}

```

**Review by martin-badin — CHANGES_REQUESTED** [review]

**Pull request closed without merging**